.github/workflows/stable.yml

```diff
@@ -0,0 +1,66 @@
+name: stable
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  build-and-release:
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - run: Write-Output VERSION="$(([Xml] (Get-Content .\src\workspacer\workspacer.csproj)).Project.PropertyGroup.Version)" >> $env:GITHUB_ENV
+    - name: Restore solution
+      run: dotnet restore
+    - name: Build
+      run: dotnet build /p:DefineConstants=BRANCH_stable --configuration Release --no-restore
```

Comment:
Since this is .Net Core can't/shouldn't we AOT-compile this using `dotnet publish` instead?

Reply:
For some reason, workspacer fails to run on `dotnet publish` for me.

Reply:
Fair enough.

Reply:
@josteink It turns out the plugins weren't copied on publish and only on build. Fixed it now.